docs/fastdds/transport/datasharing.rst

```diff
@@ -0,0 +1,213 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+
+.. _datasharing-delivery:
+
+Data-sharing delivery
+=====================
+
+*Fast DDS* allows to speed up communications between entities within the same machine
+by sharing the history of the |DataWriter| with the |DataReader| through shared memory,
+thus avoiding any of the overhead involved in the transport layer,
+and effectively avoiding any data copy between DataWriter and DataReader.
+
+.. note::
```

Comment:
Probably is useful to explain the difference between Data-sharing and Zero-Copy (maybe in Zero-Copy section).

Reply:
I added a comment about data-sharing not avoiding copies from and to the application, and referring to zero-copy. But the link to the zero-copy section cannot be created until that doc is in place.